cluster/manifests/fabric-gateway/deployment.yaml

```diff
@@ -48,6 +48,7 @@ spec:
             - -predicates=False()
             - -update-fabric-gateway-status=false
 {{ end }}
+            - -versioned-hosts-base-domain={{ .Values.hosted_zone }}
```

Comment:
does this include `ingress.cluster.local`zone?

Reply:
No

Reply:
Then we want to have a list here and include it, I guess.